.gitignore

```
@@ -16,6 +16,9 @@ temp
 packages/icons/npm
 packages/design-tokens/npm
 
+node_modules/
+*/**/node_modules/
```

We don't need this one.

```suggestion
*/**/node_modules/
```

---

Yeah I guess you're right, when I had to run it locally without pnp it added node modules to each packages folder, but I _think_ the pnp stuff might finally be mostly working for me know so it's probably fine

---

Yeah, if pnp doesn't work on a local instance - that needs to be corrected over changing the repo for the preferred scenarios where it is expected work.

---

I noticed you also removed the exclusion of the `.vscode` directory from the `.gitignore`. I put that in there because I have some extensions that add some files in there and also in different projects sometimes I like to add some scripts to a `tasks.json` file that I don't think should be included in the repo. If you want to ignore any other potential files that might be added there but keep the ones that are already there, we could add these:

```.gitignore
.vscode/*
!.vscode/extensions.json
!.vscode/launch.json
!.vscode/settings.json
```